router/server/server.go

```diff
@@ -24,6 +24,7 @@ type Server interface {
 	AddDataShard(clid string, shardKey kr.ShardKey, tsa string) error
 	UnRouteShard(sh kr.ShardKey, rule *config.FrontendRule) error
 	Datashards() []shard.Shard
+	DatashardIds() []string
 
```

Comment:
Why do we need another method in server iface? We can write separate utility function which accepts []shard.Shard and returns []string

Reply:
Add util function